### PR TITLE
Document Opensearch and Elasticsearch exporters

### LINKED ISSUE #178: Document Zeebe Elasticsearch exporter
There's very little documentation about the Zeebe Elasticsearch exporter, which is very important internally for our SREs but also for on prem users.

The Operate documentation briefly touches on it, but doesn't describe exactly how to configure it.

We should document:

1. General concept/architecture
2. Configuration
3. How to consume the exported data
4. Pitfalls/known issues

related support case https://jira.camunda.com/browse/SUPPORT-13908

### LINKED ISSUE #1828: Document Opensearch Exporter
We should With 8.2 Zeebe officially supports an Opensearch Exporter. This is already pretty well documented in the [README of this exporter in Zeebe](https://github.com/camunda/zeebe/tree/main/exporters/opensearch-exporter#zeebe-opensearch-exporter).

We should also add this documentation to the official docs. For users this is easier to find. It also provides clearer versioning, which we don't have in the README.

Besides this in the [Supported Environments](https://docs.camunda.io/docs/reference/supported-environments/) part of the docs we mention that there is no support for Opensearch. This must be removed.

Note: there is also an [open issue](https://github.com/camunda/camunda-platform-docs/issues/178) to document the Elasticsearch Exporter

### PR DESCRIPTION
**Pull request by remcowesterhoud**
## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._
The exporters are officially supported by us. Currently they are documented in READMEs in the Zeebe repo. It makes sense to have officially supported components documented in the official documentation. Because of this I have copied the contents of the READMEs to these docs.

When this is merged I will change the READMEs in the Zeebe repo to link to the official docs instead.

closes #1828 
closes #178 

Note: Even though the Elasticsearch Exporter is an older thing and is supported in previous versions, I chose not to add them to the older docs version. Reasoning behind this is that I don't know exactly which configuration option was added when. I can't copy it as-is as not all options would be available in each version.

@korthout adding you as reviewer since you reviewed my changes to the README on the Zeebe side recently. This is a 1-1 copy of those.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._
N/A

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_
With the 8.2 release

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md
Comment (korthout) [inline docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md:27]: 🔧 This note format `> **Note**` is a [specific Github notation](https://github.com/orgs/community/discussions/16925). For Docusaurus, I'd recommend switching to the [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions):

```
:::note

here's my note

:::
```

File: docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md
Comment (korthout) [inline docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md:33]: 🔧 typo: `exporter`

**Review by korthout — APPROVED** [review]
👏 This is such a valuable contribution @remcowesterhoud 🥇 

LGTM 👍 (only a few small non-blocking comments)

File: docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md
Comment (christinaausley) [reply docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md]: Adjusted this in my latest commit @remcowesterhoud 👍

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, including adding descriptions (if you want to adjust), incorporating admonitions, and adjusting to OpenSearch rather than Opensearch.

**Comment by remcowesterhoud** [comment]
@korthout I've added the retention policy here before merging which removed the stale reviews. Would you mind having a look at this and approving so I can merge?

File: docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md
Comment (korthout) [inline docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md:176]: 🔧 @remcowesterhoud  Please also add the retention example

**Review by korthout — APPROVED** [review]
Thanks @remcowesterhoud 👏 

👍 LGTM

File: docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md
Comment (remcowesterhoud) [reply docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md]: @korthout Oops my bad. Good catch, I'm gonna need a new approval now.

**Review by korthout — APPROVED** [review]

**Pull request merged**
